bin/test_client.rb

```diff
@@ -106,49 +108,54 @@ def field_should_be_nil(fields, obj, method_name)
 
 def expected_fields_in_email_resp
   %w(id
-     api_key
-     billable_units
-     to
-     subject
-     body
-     notification_type
+     reference
+     email_address
+     type
      status
-     service
      sent_at
-     sent_by
      template
-     template_version
+     body
+     subject
      created_at
-     updated_at
```

Comment:
No, but we send completed_at which is only present if the notification is delivered or failed.